src/routes/changeRequest.ts

```diff
@@ -191,6 +211,18 @@ export const confirmChangeRequest = async (req, res) => {
       break;
     case CHANGE_REQUEST_CHANGE_CARD_PIN:
       return confirmChangeCardPINHandler(req, res);
+    case CARD_TRANSACTION_CONFIRM_METHOD:
+      if (
+        changeRequestId === person.changeRequest.authenticateChangeRequestId
+      ) {
+        await confirmCardTransaction(person);
+      } else if (
+        changeRequestId === person.changeRequest.declineChangeRequestId
+      ) {
+        await declineCardTransaction(person);
+      }
```

Comment:
should we reply with err in else case?

Reply:
we can. But I will need to check the real error on sandbox